src/Orleans.Runtime/Lifecycle/SiloLifecycleStage.cs

```diff
@@ -0,0 +1,29 @@
+﻿
+namespace Orleans.Runtime
+{
+    /// <summary>
+    /// Stages of a silo's lifecycle.
+    /// </summary>
+    public enum SiloLifecycleStage
+    {
+        /// <summary>
+        /// Initialize silo runtime
+        /// </summary>
+        RuntimeInitialize = (1 << 10),
```

Comment:
I would be explicit about the bucket size using round (base 10) numbers, which are easier to troubleshoot at runtime. Say `1000` instead of `1024` (which is what `1 <<10` is)

Reply:
To this end, we could use `1e10` instead. My first impression was the same, and we have precedence with ErrorCodes to use base 10